### DISCUSSION TITLE
Mediaplayer: Volume buttons/slider configuration (feature request)

### DISCUSSION THREAD

**Discussion by Xitro01** [home-assistant/frontend#7759] [Dashboard]
I would like to request a new feature for mediaplayers.
Most of the time a TV or any other mediaplayer device is connected to a home cinema set or soundbar.
This means that the TV/mediaplayer could support a volume slider, but this will not work in combination with a sound set over CEC.

Therefore I would like to request a configuration option (e.g. volume_type: slider|buttons) that adjusts the mediaplayer’s feature set to not support a slider, although it seems to have support for it.

**Comment by zsarnett** [comment]
https://github.com/home-assistant/frontend/issues/5279

**Comment by Xitro01** [comment]
@zsarnett This does not seem to be the same feature request!
